xdk/build.gradle.kts

```diff
@@ -305,8 +305,8 @@ val installLocalDist by tasks.registering {
         // TODO: The launchers should just be application plugin scripts, this is kind of ridiculous.
         listOf("xcc", "xec", "xtc").forEach {
             val symLink = File(binDir, it)
-            logger.info("$prefix Copying launcher '$it' -> '${launcherExe.asFile}' (on Windows, this may require developer mode settings).")
-            Files.copy(launcherExe.asFile.toPath(), symLink.toPath())
+            logger.info("$prefix Creating symlink for launcher '$it' -> '${launcherExe.asFile}' (on Windows, this may require developer mode settings).")
```

Comment:
We are moving away from the symlinked installLocalDist and there are already other ways to create and run a local installation with a binary launcher. Revert this to master code to avoid unnecessary changes.